data/templates/default.latex

```diff
@@ -156,6 +156,20 @@ $if(CJKmainfont)$
   \fi
 $endif$
 \fi
+%% Support for special Unicode characters
+% Zero width non-joiner
+\def\zerowidthnonjoiner{%
+  \texorpdfstring{%
+    \textormath{\nobreak\discretionary{-}{}{\kern.03em}%
+      \ifvmode\else\nobreak\hskip\skip11\fi}{}%
```

Comment:
[Sorry for the new round of comments]

I'm not sure that `\skip11` is more readable than `\z@skip`.  Is this an intentional change in comparison to the code from babel-german?  If yes, wouldn't it be better to give `\skip11` a name?

Reply:
Agreed. The `\skip11` came from trying to avoid having to use the verbose `\makeatletter`/`\makeatother` combination which I, personally, find quite annoying. OTHO, `\skip11` is *really* cryptic, so let's go for the more verbose variant.

Reply:
You don't have to use `\makeatletter` and `\makeatother`.  Simply use `0pt plus0pt minus0pt` instead of `\skip11`.